examples/anchor_text_movie.ipynb

```diff
@@ -17,18 +17,29 @@
   {
```

Comment:
<p>Can you set the kernel to default Python 3 before saving please? The reason is that if whoever runs the code and doesn't have an <code>alibi</code>  environment will get a warning message when loading the notebook that a kernel could not be found.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>